local-setup/jupyter/crs_loader.py

```
@@ -0,0 +1,1363 @@
+    def _delete_boundaries_via_db(self, tenant: str) -> Dict:
+        """Delete boundaries using direct database access (requires kubectl)"""
+        import subprocess
+
+        # Database connection details
+        db_host = "chakshu-pgr-db.czvokiourya9.ap-south-1.rds.amazonaws.com"
+        db_name = "chakshupgrdb"
+        db_user = "chakshupgr"
+
+        # Get DB password from K8s secret
+        pw_result = subprocess.run(
+            ["kubectl", "get", "secret", "db", "-n", "egov", "-o", "jsonpath={.data.password}"],
```

Ruff (0.15.0)

[error] Starting a process with a partial executable path

(S607)